larpandoracontent/LArTrackShowerId/SvmPfoCharacterisationAlgorithm.cc

```diff
@@ -9,16 +9,31 @@
 
 #include "larpandoracontent/LArHelpers/LArFileHelper.h"
 #include "larpandoracontent/LArHelpers/LArMCParticleHelper.h"
+#include "larpandoracontent/LArHelpers/LArMonitoringHelper.h"
+#include "larpandoracontent/LArHelpers/LArClusterHelper.h"
 #include "larpandoracontent/LArHelpers/LArPfoHelper.h"
 #include "larpandoracontent/LArHelpers/LArMvaHelper.h"
-
+#include <vector>
+#include "larpandoracontent/LArHelpers/LArInteractionTypeHelper.h"
+#include "Helpers/MCParticleHelper.h"
+#include "larpandoracontent/LArObjects/LArTwoDSlidingFitResult.h"
 #include "larpandoracontent/LArTrackShowerId/SvmPfoCharacterisationAlgorithm.h"
+#include "larpandoracontent/LArHelpers/LArGeometryHelper.h"
+#include "larpandoracontent/LArTrackShowerId/CutClusterCharacterisationAlgorithm.h"
+#include "larpandoracontent/LArTrackShowerId/CutPfoCharacterisationAlgorithm.h"
 
 using namespace pandora;
 
 namespace lar_content
 {
-
+    SvmPfoCharacterisationAlgorithm::RecoParameters::RecoParameters() :
```

Comment:
Not sure what to do here either. While this is deriving from Isobel's unfolded pfo hierarchy code, i can't just remove this since it is used in my implementation as well. Also, not sure what you mean by "entire file/class should be renamed ..." Please clarify a bit more. Thank you.

Reply:
In this case I'm referring to the fact that the algorithm is called SvmPfoCharacterisationAlgorithm, but it's now based on a BDT rather than an SVM, right? The name should probably change to reflect that.

Reply:
Ah ok. I don't know if I have the authority to change the name of this algorithm. Maybe @johnmarshall80 could chime in on this. BdtPfoCharacterisationAlgorithm is probably better.